lib/libutils/isoc/bget_malloc.c

```diff
@@ -851,7 +862,7 @@ void nex_free(void *ptr)
 {
 	uint32_t exceptions = malloc_lock(&nex_malloc_ctx);
 
-	raw_free(ptr, &nex_malloc_ctx);
+	raw_free(ptr, &nex_malloc_ctx, false /* wipe */);
```

Comment:
I think the `/* wipe */` comment is unnecessary and also misleading, as read now, I can interpret it as either "this is the wipe variable" or "false means wipe". I think it's better to get rid of it, we never (?) document variables like this otherwise.

Reply:
I usually write like this too. As a reminder of the name of the function parameter, but when the value is `false` I put a `!` in front of the name of the parameter in the comment. With a `!` added here too I think there would be no confusion.

Reply:
I quite like `/* !wipe */`. @jbech-linaro what do you say?

Reply:
I'd prefer not using it, but not worth argue about it, so please go ahead.